SPECS/ucx/ucx.spec

```diff
@@ -8,23 +8,28 @@
 %bcond_with    rocm
 %bcond_with    ugni
 %bcond_with    xpmem
+%bcond_with    vfs
 
 Summary:        UCX is a communication library implementing high-performance messaging
 Name:           ucx
-Version:        1.11.0
-Release:        4%{?dist}
+Version:        1.15.0
+Release:        1%{?dist}
 License:        BSD
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System Environment/Security
 URL:            http://www.openucx.org
-Source0:        https://github.com/openucx/%{name}/releases/download/v1.11.0/ucx-1.11.0.tar.gz
+Source0:        https://github.com/openucx/%{name}/releases/download/v1.15.0/ucx-1.15.0.tar.gz
 
 
 # UCX currently supports only the following architectures
 ExclusiveArch: aarch64 ppc64le x86_64
 
-BuildRequires: automake autoconf libtool gcc-c++ numactl-devel
+%if %{defined extra_deps}
+Requires: %{?extra_deps}
+%endif
+
+BuildRequires: automake autoconf libtool gcc-c++
 %if %{with cma}
 BuildRequires: glibc-devel >= 2.15
 %endif
```

Comment:
Can we also get rid of the dependency on `knem` (which is currently not buildable on 3.0-dev?